.github/workflows/dusk_ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2020-10-25
+          toolchain: nightly-2021-04-23
```

Comment:
Should not be updated unless we do so for all of our repos.

Reply:
Replied in RC with the current version `2021-04-23`

Reply:
I saw. I already said it was a bad call from my  side. The only times when this should be changed is either we do it everywhere or if we use `resolver2` feature and the toolchain does not support it.

Reply:
I think we agreed to do it everywhere - so this one is for poseidon.